File: python/image_upload_nsfw.py
import requests
import json

# Upload an image first & detect whether it is Not Suitable for Work (NSFW)
# https://pixlab.io/#/cmd?id=nsfw

req = requests.post('https://api.pixlab.io/nsfw',
	files = {'file': open('local_image.png', 'rb')},
	data={
		'key':'PIXLAB_API_KEY', # Get your API key from the console at: https://console.pixlab.io
	}
)
reply = req.json()
if reply['status'] != 200:
	print (reply['error'])
else:
	print ("NSFW Score: "+ str(reply['score']))

File: PHP/image_upload_nsfw.php

<?php
/*
 * PixLab PHP Client which is just a single class PHP file without any dependency that you can get from Github
 * https://github.com/symisc/pixlab-php 
 */
require_once "pixlab.php";

# Upload an image first & detect whether it is Not Suitable for Work (NSFW)
# https://pixlab.io/#/cmd?id=nsfw

# Your PixLab key
$key = 'My_Key';

/* Path for the image to upload */
$img_path = './local_img.png';

/* Process */
$pix = new Pixlab($key);
if( !$pix->post('nsfw',array('key' => $key), $img_path) ){
	echo $pix->get_error_message()."\n";
	die;
}
/* Grab NSFW score */
echo "NSFW score: ".$pix->json->score."\n";

?>
